### PR TITLE
Create in engine.go renamed to CreateContainer

### PR DESCRIPTION
**Pull request by doronp**
Signed-off-by: Doron Podoleanu <doronp@il.ibm.com>

### CODE REVIEW

**Comment by nishanttotla** [comment]
LGTM

This helps with clarity of code. Ping @allencloud

**Comment by allencloud** [comment]
Yes. @nishanttotla 
Jenkins failure is unrelated.

LGTM

File: cluster/engine.go [hunk 1]
```diff
@@ -862,7 +862,7 @@ func (e *Engine) TotalCpus() int64 {
 }
 
 // Create a new container
```
Comment (allencloud) [inline cluster/engine.go]: Can you change the annotation to `CreateContainer creates a new container` ?

**Comment by allencloud** [comment]
Hi, @doronp 
Can you rebase the code and solve the conflict here? When you have solved this and changed the function annotation, we are going to merge this.

**Comment by nishanttotla** [comment]
Failures seem unrelated to this PR. Ping @doronp, please update the comment as @allencloud suggested.

**Comment by allencloud** [comment]
@doronp 
Thanks for your contribution. Just One more thing, could you squash the two commits into a single one?
Still LGTM.

**Comment by nishanttotla** [comment]
I did a squash-merge. Looks like it doesn't create a merge commit though, we should avoid doing that in the future.

**Pull request merged**